### PR TITLE
falcon-scripts improvements

### PR DESCRIPTION
**Pull request by tutok**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
attempt to solve: #490

**What is the current behavior? (You can also link to an open issue here)**
n/a

**What is the new behavior (if this is a feature change)?**
n/a

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**
n/a

**Other information:**
n/a

### CODE REVIEW

File: packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js [hunk 1]
```diff
@@ -53,3 +54,50 @@ module.exports = async ({ packagePath }) => {
...
+module.exports.cli = async ({ packagePath }) => {
+  console.log('building cli...');
+
+  process.env.ROLLUP = true;
+  // process.env.TARGET = 'NODE';
+
+  // eslint-disable-next-line
+  const packageJson = require(path.join(packagePath, './package.json'));
+
+  const directory = path.join('src', 'cli');
```
Comment (Borales) [inline packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js]: @tutok just to keep these things unified - what about using `src/bin` as a folder name?

Comment (tutok) [reply packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js]: 👍

File: packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js [hunk 2]
```diff
@@ -53,3 +54,50 @@ module.exports = async ({ packagePath }) => {
   const bundle = await rollup.rollup(inputOptions);
   await bundle.write(outputOptions);
 };
+
+module.exports.cli = async ({ packagePath }) => {
+  console.log('building cli...');
+
+  process.env.ROLLUP = true;
+  // process.env.TARGET = 'NODE';
```
[the remark this answers is not in the document]
Comment (tutok) [reply packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js]: done

File: packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js [hunk 3]
```diff
@@ -14,10 +14,11 @@ const makeExternalPredicate = externalsArr => {
   return id => externalPattern.test(id);
 };
 
-module.exports = async ({ packagePath }) => {
+module.exports.pkg = async ({ packagePath }) => {
   Logger.log('building cjs...');
 
   process.env.ROLLUP = true;
+  // process.env.TARGET = 'NODE';
```
[the remark this answers is not in the document]
Comment (tutok) [reply packages/falcon-dev-tools/falcon-scripts/src/build-cjs.js]: done

**Pull request merged**